anta/tests/configuration.py

```diff
@@ -94,9 +94,9 @@ class VerifyRunningConfigLines(AntaTest):
     ```yaml
     anta.tests.configuration:
       - VerifyRunningConfigLines:
-            regex_patterns:
-                - "^enable password.*$"
-                - "bla bla"
+          regex_patterns:
+            - "^enable password.*$"
+            - "bla bla"
```

Comment:
Does the script still work if the indentation is not ok?

Reply:
no

Reply:
That's bad no? That means a dev can create a new test with wrong indentation, pre-commit will not add it, we will forget and the world is sad.

Reply:
we are just printing what is in the Example - if the Example is bad then we print bad stuff and it is a bug. I have fixed what I could but that won't save us from wrong indentation without extra work thats not really worth it